reconcile/test/test_slack_usergroups.py

```diff
@@ -86,6 +89,35 @@ def test_get_pagerduty_username_slack_username(self):
         result = integ.get_pagerduty_name(user)
         self.assertEqual(result, 'pd')
 
+    @patch.object(integ, 'SlackApi', autospec=True)
+    @patch.object(queries, 'get_app_interface_settings', autospec=True)
+    @patch.object(queries, 'get_permissions', autospec=True)
+    def test_get_slack_map_return_expected(self, get_permissions,
```

Comment:
nit: calling these `mock_<object_name>` makes things a bit clearer if testing methods get bigger. Maybe we can get that documented as a convention in the future, but you'll see the Python docs do similar.